OpenTrons/opentrons-api
# refactor(app): create a module deck map in calibrate page

# Overview
This PR displays the module deck map when you click on the module list on the side-panel on the calibrate page. See design [here](https://www.figma.com/file/GWe9EvcpyehHSAb4bfEKrK/Multiples-of-a-Module-(MoaM)?node-id=291%3A3033)


# Changelog
- combined all modules into one `ListItem` on the side panel
- updated Module component on the deck map to reflect the new design
- updated tests


# Review requests
Test on robot/simulator
- Upload a protocol that requires multiples of the same module model
- Click on the module lists to see the new module deck map
- The deck should show correctly whether or not the expected module is connected
- Port info should be N/A (not implemented)

TODO: make sure close thermocycler lid modal still shows up before moving on to labware calibration



# Risk assessment
Medium, this PR would affect the pre-protocol calibration flow for all user. This branch should be be tested thoroughly before merging.
